FileManager/Services/KeychainService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  KeychainService.swift
+//  FileManager
+//
+//  Created by Екатерина Неделько on 14.06.22.
+//
+
+import KeychainSwift
+
+class KeychainService {
+    
+    // MARK: - Public Properties
+    
+    static let shared = KeychainService()
+    
+    // MARK: - Private Properties
+    
+    private let keychain = KeychainSwift()
+    
+    private let keychainKey = "keychainKey"
+    
+    // MARK: - Initialization
+    
+    private init() { }
+    
+    // MARK: - Keychain methods
+    
+    func getKeychain() -> String? {
+        keychain.get(keychainKey)
+    }
+    
+    func setKeychain(value: String) {
+        keychain.set(value, forKey: keychainKey)
+    }
```

Comment:
Shouldn't you pass key to these methods? Or you should rename your methods to understand what value you work with